core/backend/src/IModelHost.ts

```diff
@@ -571,6 +571,17 @@ export class IModelHost {
       throw new IModelError(BentleyStatus.ERROR, "Cannot use both Azure and custom cloud storage providers for tile cache.");
     }
   }
+
+  /**
+   * Generate SHA1 Hash of Schema XML
+   * @param schemaXmlPath: Path where schema XML file is located
+   * @param referencePaths: Schema reference paths
+   * @param exactMatch: Schema references are located by exact scheme version comparisons
+   * @internal
+   */
+  public static computeSchemaChecksum(arg: { schemaXmlPath: string, referencePaths: string[], exactMatch?: boolean }): string {
```

Comment:
This documentation should be in `NativeLibrary.ts` in the other PR, not here.

Reply:
Ok let me remove it from here